packages/flutter/lib/src/painting/border_radius.dart

```diff
@@ -395,10 +395,10 @@ class BorderRadius extends BorderRadiusGeometry {
   RRect toRRect(Rect rect) {
     return RRect.fromRectAndCorners(
       rect,
-      topLeft: topLeft,
-      topRight: topRight,
-      bottomLeft: bottomLeft,
-      bottomRight: bottomRight,
+      topLeft: topLeft.clamp(minimum: Radius.zero), // ignore_clamp_double_lint
+      topRight: topRight.clamp(minimum: Radius.zero), // ignore_clamp_double_lint
+      bottomLeft: bottomLeft.clamp(minimum: Radius.zero), // ignore_clamp_double_lint
+      bottomRight: bottomRight.clamp(minimum: Radius.zero), // ignore_clamp_double_lint
```

Comment:
Are these ignores temporary? If so, maybe add a TODO?

Reply:
Also, why are these ignores needed in the first place? You aren't even calling clamp on double here.

Is the `ignore_clamp_double_lint` triggering too broadly and needs some tightening?

Reply:
Yes, I wondered the same thing, but there are reasons why we can't narrow it.

See [here](https://github.com/flutter/flutter/blob/278637748bf9088c660ef495804e56c557299d59/dev/bots/analyze.dart#L242).